### ISSUE TITLE
Manual formatting

### ISSUE THREAD

**Issue by paulduchesne** [FIAF/fiaf-cataloguing-manual#1]
@heftberger @natashafairbairn place to discuss markdown formatting options.

**Comment by paulduchesne** [comment]
Two possible ways to indicate "examples" would be either "blockquotes" or "code blocks".

~

Different footage used from the original realisation of the Work, even where
footage used was shot at the same time and edited together by the same
people, or following the pattern set by original creators.

> **The rules of the game (1939)** exists only as a Variant of the original Work,
made partially from alternate takes to those used in the original release. 

Different footage used from the original realisation of the Work, even where
footage used was shot at the same time and edited together by the same
people, or following the pattern set by original creators.

```
The rules of the game (1939): exists only as a Variant of the original Work,
made partially from alternate takes to those used in the original release. 
```

~

I would favour blockquotes slightly, as they allow additional formatting and take care of linebreaks automatically.

**Comment by natashafairbairn** [comment]
Hi Paul,

If blockquotes is this one:

The rules of the game (1939) exists only as a Variant of the original Work,
made partially from alternate takes to those used in the original release.

Then that's my preference - it's a bit more aesthetically pleasing too.

Are you able to create a quick test document for me in Github (just a page/section from the manual will be fine) so that I can have an experimental play about familiarising myself rather than mess up any actual Manual sections? I wasn't quite sure how to do that and didn't want to mess up anything in the FIAF Github sections.

Thanks,
Natasha
________________________________

**Comment by paulduchesne** [comment]
Closed as issue has also been addressed in #13
